### PR TITLE
prepare blockchain test suites

### PR DESCRIPTION
**Pull request by winsvega**

### CODE REVIEW

File: retesteth/Socket.cpp [hunk 1]
```diff
@@ -136,20 +164,28 @@ namespace
...
             curl_easy_setopt(curl, CURLOPT_URL, url.c_str());
             curl_easy_setopt(curl, CURLOPT_BUFFERSIZE, 3000000);
             curl_easy_setopt(curl, CURLOPT_WRITEFUNCTION, writecallback);
             curl_easy_setopt(curl, CURLOPT_WRITEDATA, httpData.get());
+            curl_easy_setopt(curl, CURLOPT_READFUNCTION, readcallback);
+            curl_easy_setopt(curl, CURLOPT_READDATA, &wt);
+            curl_easy_setopt(curl, CURLOPT_POST, 1L);
+
             struct curl_slist *header = NULL;
+            header = curl_slist_append(header, "Accept: text/plain");
```
Comment (shemnon) [inline retesteth/Socket.cpp]: Should this accept applicaiton/json instead?  Or both?  This currently breaks us.

File: retesteth/RPCSession.cpp [hunk 2]
```diff
@@ -264,30 +264,19 @@ string RPCSession::eth_getCode(string const& _address, string const& _blockNumbe
...
-
-string RPCSession::eth_sendTransaction(TransactionData const& _td)
-{
-	return rpcCall("eth_sendTransaction", { _td.toJson() }).asString();
-}
-
-string RPCSession::eth_call(TransactionData const& _td, string const& _blockNumber)
-{
-	return rpcCall("eth_call", { _td.toJson(), quote(_blockNumber) }).asString();
+    return rpcCall("eth_blockNumber", {}).asInt();
```
Comment (shemnon) [inline retesteth/RPCSession.cpp]: Ethereum JSON-RPC standard is a hex encoded string (https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_blocknumber) - any reason this should vary from the standard?

Comment (winsvega) [reply retesteth/RPCSession.cpp]: hm, looks like I get integer from geth

Comment (shemnon) [reply retesteth/RPCSession.cpp]: The spec says string.  However it was coded before worked.

**Pull request merged**